helpfunctions/utils.py

```diff
@@ -207,17 +207,21 @@ def getLeaderboardPageBy(self, page, sortBy):
                 # When user is not in guild.
                 nick = "Not on guild"
                 name = f"ID: {userID}"
+            # Check length of nick + name
+            nick = nick if len(nick) <= 12 else nick[:-3] + "..."
+            name = name if len(name) <= 22 else name[:-3] + "..."

```

Comment:
When the nick is for example `0123456789abcdefgh` then nick will be `0123456789abcde...` instead of `0123456789ab...`.
Only the nicks last characters will be replaced by `...` and its length will be the same, but it should be shortened.